src/server/controller/area/updateCountry.ts

```diff
@@ -34,8 +34,12 @@ export const updateCountry = async (props: Props, client: BaseProtocol = DB): Pr
       t
     )
 
-    // update cache
-    await AreaRedisRepository.getOneCountry({ assessment, cycle, countryIso, force: true }, t)
+    // update cache for all cycles to keep lastPublished up to date
+    await Promise.all(
+      assessment.cycles.map((c) =>
+        AreaRedisRepository.getOneCountry({ assessment, cycle: c, countryIso, force: true }, t)
+      )
+    )
```

Comment:
When we update country (status) we need to update all cycle's caches to be consistent with lastPublished information in cache